include/network.php

```diff
@@ -4,6 +4,9 @@
  * @file include/network.php
  */
 
+use \Friendica\Core\Config;
+use \Friendica\Core\PConfig;
```

Comment:
`use \Friendica\Core\PConfig;` isn't really needed here because you only use the config (not the pconfig) class in the code.